src/main/java/org/broadinstitute/hellbender/utils/SATagAlignment.java

```diff
@@ -0,0 +1,122 @@
+package org.broadinstitute.hellbender.utils;
+
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Class for parsing and storing the information about supplementary alignments contained in a read's "SA" attribute.
+ */
+public class SATagAlignment {
```

Comment:
Steve's comments below add up to suggesting that this class be an immutable with natural types for its fields.  I agree.